apps/librepcb/main.cpp

```diff
@@ -205,7 +205,22 @@ static bool isFileFormatStableOrAcceptUnstable() noexcept {
  ******************************************************************************/
 
 static FilePath determineWorkspacePath() noexcept {
-  FilePath wsPath(Workspace::getMostRecentlyUsedWorkspacePath());
+  static const QString FORCE_VAR_NAME("LIBREPCB_WORKSPACE");
```

Comment:
Couldn't we use a plain C string here, to avoid the conversion `.toStdString().c_str()` below? And since the env var is no longer named "force", it would be nice to rename the variable as well ;-)

Something like this:

```suggestion
  const char* workspaceEnvVarName = "LIBREPCB_WORKSPACE";
```

I'm not sure if needed, but if `qWarning()` below really needs a `QString`, the conversion can be done easily with `QString(workspaceEnvVarName)`.